README.md

```diff
@@ -26,7 +26,7 @@ Tools for working with the [SceneNet RGB-D](https://robotvault.bitbucket.io/scen
 
     ```bash
     cd ../scenenet_ros_tools && chmod +x nodes/scenenet_to_rosbag.py
-    rosrun scenenet_ros_tools scenenet_to_rosbag.py -scenenet_path PATH/TO/pySceneNetRGBD -trajectory TRAJECTORY -to_frame TO_FRAME -output_bag OUTPUT_BAG
+    rosrun scenenet_ros_tools scenenet_to_rosbag.py -scenenet_path PATH/TO/pySceneNetRGBD -trajectory TRAJECTORY -to_frame TO_FRAME -output_bag OUTPUT_BAG -protobuf_path PATH/TO/protobuf_file -dataset_type EITHER_val_OR_train
```

Comment:
Can't we infer the protobuf_file path from the dataset_type? We don't want to allow the user to specify train dataset type and the val protobuf. 
Also, my personal preference is to leave detailed explanation of the valid range of values to the --help strings of the arguments. 
Finally, add an example usage of any new argument to the sample command on line 32.

```suggestion
    rosrun scenenet_ros_tools scenenet_to_rosbag.py -scenenet_path PATH/TO/pySceneNetRGBD -trajectory TRAJECTORY -to_frame TO_FRAME -output_bag OUTPUT_BAG -dataset_type DATASET_TYPE
```

Reply:
I apologize for the very late reply. The problem with automatically finding the path of the protobuf file is that the training dataset is split into 17 smaller sets, and therefore there are 17 protobuf files with filenames `scenenet_rgbd_train_NUM.pb`, where `NUM` is between 0 and 16. One solution could be instructing the user (commit d7436dc985ac2f09b4288c12a7b3af11116157ce) to extract all the protobuf file from the tar archive (http://www.doc.ic.ac.uk/~ahanda/train_protobufs.tar.gz) in the `pySceneNetRGBD/data` folder, as already done for the validation protobuf `scenenet_rgbd_val.pb`. Then, one could refer to each training set with a different `dataset_type`, e.g., `train_0`, `train_1`, etc. and let `scenenet_to_rosbag.py` automatically find the `protobuf_path` as at line 437 of 3aa93ce26c031f252925fc88f09be3bb8ccdd667. This would remove the need for the argument `protobuf_path`.